inlang/source-code/paraglide/paraglide-js/package.json

```diff
@@ -41,20 +41,23 @@
 	"dependencies": {
 		"@inlang/detect-json-formatting": "1.0.0",
 		"@inlang/sdk": "*",
+		"@inlang/telemetry": "*",
+		"@inlang/env-variables": "*",
```

Comment:
1. @inlang/env-variables is a private package. you can't use it in dependencies
2. @inlang/telemetry is also a private package

I recommend removing both dependencies and hardcode the endpoint + public access key to push events. that should be sufficient.

Reply:
> Just to check, are you aware that @inlang/cli has those same dependencies?

Yes. The CLI is bundled. Thus, the issue does not apply there (we could also bundle the paraglide cli)

Reply:
I believe bundling is the way to go here. If we hardcode the token there are some serious drawbacks:
- No separation between dev & production tokens
- Almost guaranteed to be forgotten if the posthog token ever changes

Reply:
Do you want to add bundling in this PR? 

Btw, please only the CLI. Don't bundle the "library".